plugins/inputs/opcua/README.md

```diff
@@ -89,6 +89,11 @@ Plugin minimum tested version: 1.16
   #  {name="", namespace="", identifier_type="", identifier=""},
   #  {name="", namespace="", identifier_type="", identifier=""},
   #]
+  
+  ## Enable workarounds required by some devices to work correctly
+  # [inputs.opcua.workarounds]
+  ## Set valid status codes
+  # valid_status_codes = ["0x0", "0xC0"]
```

Comment:
The commented setting shows the default and it's two values here. In init() in opcua_client.go, the status is set to default only to statusOK.
```suggestion
  # valid_status_codes = ["0x0"]
```
When we change this we also need to update the sampleConfig variable in opcua_client.go.